test/test_serialization.py

```diff
@@ -4155,9 +4156,13 @@ def test_serialization_dtype(self, dtype, weights_only):
             y = torch.load(f, weights_only=weights_only)
             self.assertEqual(y['x'], x)
             # Check that views are actually views
-            y['odd'][0] = torch.tensor(0.25, dtype=dtype)
-            y['even'][0] = torch.tensor(-0.25, dtype=dtype)
-            self.assertEqual(y['x'][:2].to(dtype=torch.float32), torch.tensor([-0.25, 0.25]))
+            if dtype in (torch.uint16, torch.uint32, torch.uint64):
```

Comment:
```suggestion
            if not dtype.is_signed:
```
And flip the order to remove the "not" as well I guess